packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/JSBundleLoader.kt

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package com.facebook.react.bridge
+
+import android.content.Context
+import com.facebook.infer.annotation.Nullsafe
+import com.facebook.react.common.DebugServerException
+
+/** A class that stores JS bundle information and allows a [JSBundleLoaderDelegate]. */
+@Nullsafe(Nullsafe.Mode.LOCAL)
+public abstract class JSBundleLoader {
+
+  /** Loads the script, returning the URL of the source it loaded. */
+  public abstract fun loadScript(delegate: JSBundleLoaderDelegate): String
+
+  public companion object {
```

Comment:
Can you move this to the bottom of the file? more Kotlin idiomatic.

Reply:
The companion object is already at it contains all the methods which are previously static in java.

Reply:
Ah you are totally right, I missed it!